src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/AbstractRabbitSubscriber.java

```diff
@@ -150,6 +148,10 @@ public void start() throws Exception {
                         LOGGER.info("Start listening queue name='{}'", queue);
                     } catch (IOException e) {
                         throw new IllegalStateException("Can not start subscribe to queue = " + queue, e);
+                    } catch (InterruptedException e) {
+                        Thread.currentThread().interrupt();
+                        LOGGER.error("Interrupted exception while consuming from queue '{}'", queue);
+                        throw new IllegalStateException("Thread was interrupted while consuming", e);
```

Comment:
Please add throws InterruptedException to the method

Reply:
The exception is thrown inside a callback, and it won't leave scope of the callback